packages/shared/modals/ManagedResourceSaveModal.tsx

```diff
@@ -0,0 +1,80 @@
+import * as React from 'react';
+import {
+  ResourceLink,
+  K8sResourceCommon,
+  OwnerReference,
+} from '@openshift-console/dynamic-plugin-sdk';
+import { TFunction } from 'i18next';
+import { Trans } from 'react-i18next';
+import {
+  Button,
+  Modal,
+  ModalVariant,
+  ButtonVariant,
+} from '@patternfly/react-core';
+import { ExclamationTriangleIcon } from '@patternfly/react-icons';
+import { useCustomTranslation } from '../useCustomTranslationHook';
+import { referenceForOwnerRef } from '../utils';
+import { ModalBody, ModalFooter, ModalHeader, CommonModalProps } from './Modal';
+
+const header = (t: TFunction) => (
+  <ModalHeader>
+    <ExclamationTriangleIcon color="#f0ab00" className="icon--spacer" />
```

Comment:
```suggestion
    <ExclamationTriangleIcon color="#f0ab00" className="icon--spacer" />
```
Do not use magic strings. Use PF var references. This will cause problems with themes.